modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -66,9 +66,10 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
         // =======================================================
 
         if (!empty($scan_id_types)) {
-            $acqpid       = "AND AcquisitionProtocolID IN ("
-                . implode(",", array_keys($scan_id_types))
-                . ")";
+            // it seems like a unused local variables by phan check
+            //            $acqpid       = "AND AcquisitionProtocolID IN ("
```

Comment:
If it's unused it should be deleted, not commented out.. @cmadjar do you know why this would be here but never used? I don't see any references to the variable in the code when I ctrl-f in the file on GitHub.